python/frontend/cirrus/cirrus/utils.py

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Remove this spaces.

Reply:
I removed the extra spaces, though in python proper style says there should be 2 new lines between functions